src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1982,7 +1982,13 @@ private SymWithType GetSlotForOverride(MethodInfo method)
 
                 AggregateSymbol aggregate = GetCTypeFromType(baseMethodInfo.DeclaringType).getAggregate();
                 MethodSymbol baseMethod = FindMethodFromMemberInfo(baseMethodInfo);
-                Debug.Assert(baseMethod != null);
+
+                // This assert is temporarily disabled to improve testability of the area on .NetNative
```

Comment:
Does anyone know if this temporary disabling is still necessary? Does anyone know what issue it related to? It should hold, and the branch below should be dead code that could be removed.